modules/mesh-layers/package.json

```diff
@@ -33,6 +33,7 @@
     "@deck.gl/core": "^7.0.0"
   },
   "dependencies": {
+    "@luma.gl/addons": "^7.2.0-alpha.1",
```

Comment:
Can this be 7.10? I don't like the alpha part.

Reply:
It might cause 2 versions of luma to be installed.

Reply:
Core is also including `alpha.1`